MediaBrowser.Providers/Subtitles/SubtitleManager.cs

```diff
@@ -48,9 +48,7 @@ public class SubtitleManager : ISubtitleManager
             _monitor = monitor;
             _mediaSourceManager = mediaSourceManager;
             _localization = localizationManager;
-            _subtitleProviders = subtitleProviders
-                .OrderBy(i => i is IHasOrder hasOrder ? hasOrder.Order : 0)
-                .ToArray();
+            _subtitleProviders = [.. subtitleProviders.OrderBy(i => i is IHasOrder hasOrder ? hasOrder.Order : 0)];
```

Comment:
Keep the `ToArray`
https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA+ABATARgLABQGADAAQY4oDchJ5OAdADICWAdgI41EDM5WpAMKkA3oVITyfCsQDaAXVIBZABSsAzgBcAPDIB8pTQEpR4yeYwB2UrIYNNDAOoALGLBUJSAXgMIGgiDZNAEN2dRUAIk0YLQijI3lucwBfM3N0jMyJNIxpHDlFJSw1Fi1dfINjUwIsq0MnV3dPH1I/AKDQtnComM04owYAFQgAQSgoYIBPFSMkyVSCZKA==